### ISSUE TITLE
OOI selector for config is limited

### ISSUE THREAD

**Issue by zcrt** [minvws/nl-kat-coordination#1441]
**Describe the bug**
When using `/en/[organisation]/objects/add/Config/` one has to select an OOI. However this selector seems to be limited to an _n_ amount of objects. However, as I understand it, it should be possible to add a Config object to any object.

**To Reproduce**
1. Create an organisation with multiple objects'
2. Go to https://kat.dev-kat.z-cert.lan/en/<organisation>/objects/add/Config/
3. Notice not all your objects are in the selector.

**Expected behavior**
All possible objects in the selector, or searchable in the selector.

**OpenKAT version**
1.10.0rc2

**Comment by underdarknl** [comment]
Does this also manifest when you try to add an object when using the add Object button present on the related object you wanted to select?

Eg, go to Network|Internet, add object configOOI, then the object where you came from should be in the list, unrelated to the limit.

**Comment by zcrt** [comment]
> Does this also manifest when you try to add an object when using the add Object button present on the related object you wanted to select?
> 
> Eg, go to Network|Internet, add object configOOI, then the object where you came from should be in the list, unrelated to the limit.

No, then it does not manifest. This seems to be a working workaround, thanks!
Nevertheless, it would be nice to be able to select any object in the selector, but it has a lower priority then
